airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py

```diff
@@ -32,4 +32,4 @@ def stream_slices(self) -> Iterable[StreamSlice]:
         else:
             for parent_stream_config in self.parent_stream_configs:
                 stream_state_field = parent_stream_config.partition_field.eval(self.config)
-                yield {stream_state_field: custom_stream_state_value, "parent_slice": {}}
+                yield StreamSlice(partition={stream_state_field: custom_stream_state_value, "parent_slice": {}}, cursor_slice={})
```

Comment:
very nit: I find it a bit awkward that when reviewing this PR we have no actual clue that the CDK version was updated which likely leads to this change.

Reply:
what information would make it easier to review?

Reply:
Nothing, it's just that `^0` is not very explicit, but it makes sense to use it.